src/networks.js

```diff
@@ -16,3 +16,20 @@ export const networks = {
   },
   main: { court: '0xee4650cBe7a2B23701D416f58b41D8B76b617797' },
 }
+
+export const networkConfigs = {
+  rpc: {
+    nodes: {
+      defaultEth: 'http://localhost:8545',
+    },
+  },
+  main: {
+    nodes: {
+      defaultEth: 'wss://mainnet.eth.aragon.network/ws',
```

Comment:
It turns out [Ethers.js didn't have support for websockets until last week](https://github.com/ethers-io/ethers.js/issues/141#issuecomment-598686017), and this is still in its v5 beta.

For now, since we don't need events or need to query that often, let's continue using the http endpoint:

```suggestion
      defaultEth: 'https://mainnet.eth.aragon.network/',
```